docs/full-node/devnet.md

```diff
@@ -0,0 +1,94 @@
+---
+sidebar_position: 50
+title: Devnet
+sidebar_label: Devnet
+---
+
+import UntranslatedPageText from "@site/src/components/UntranslatedPageText";
+
+<UntranslatedPageText />
+
+# Create a local Devnet
+
+## Using Docker
+
+```sh
+npx @alephium/cli@latest devnet start
+```
+
+This launch a devnet with [this configuration](https://github.com/alephium/alephium-web3/blob/master/packages/cli/devnet-user.conf)
+
+The Typescript SDK is then able to interact with the network through REST endpoints.
+
+Alternatively, if you want to create a local development network with explorer support, please use `docker-compose` and follow the instructions in [alphium-stack](https://github.com/alephium/alephium-stack#devnet).
```

Comment:
Maybe we can have this at the end of this page (after explorer backend) since it will create both full node and explorer backend.

Reply:
what about an third section `Other alternatives` after the `Manually using jar files` one?

Reply:
I push a proposition in my latest commit, let me know.